### PR TITLE
[4.0] Language Install button [a11y]

### PR DESCRIPTION
**Pull request by brianteeman**
On the install languages page every language has an identically named button. Just as you wouldnt have more than one link with the same name you shouldnt have more than one button.

This PR adds an aria-label to each button with the value of "Install <language name>"

To test apply pr and view source and you will see the aria-label
image

(cc @carcam as agreed but you dont seem to be replying)

### CODE REVIEW

File: administrator/components/com_installer/tmpl/languages/default.php [hunk 1]
```diff
@@ -69,7 +69,12 @@
 									<?php $buttonText = (isset($this->installedLang[0][$language->code]) || isset($this->installedLang[1][$language->code])) ? 'REINSTALL' : 'INSTALL'; ?>
 									<?php $buttonClass = (isset($this->installedLang[0][$language->code]) || isset($this->installedLang[1][$language->code])) ? 'btn btn-success btn-sm' : 'btn btn-primary btn-sm'; ?>
 									<?php $onclick = 'document.getElementById(\'install_url\').value = \'' . $language->detailsurl . '\'; Joomla.submitbutton(\'install.install\');'; ?>
-									<input type="button" class="<?php echo $buttonClass; ?>" value="<?php echo Text::_('COM_INSTALLER_' . $buttonText . '_BUTTON'); ?>" onclick="<?php echo $onclick; ?>">
+									<input type="button"
+										class="<?php echo $buttonClass; ?>"
+										value="<?php echo Text::_('COM_INSTALLER_' . $buttonText . '_BUTTON'); ?>"
+										aria-label="<?php echo Text::_('COM_INSTALLER_' . $buttonText . '_BUTTON') . " " . $language->name; ?>"
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/languages/default.php]:
```suggestion
										aria-label="<?php echo Text::_('COM_INSTALLER_' . $buttonText . '_BUTTON') . ' ' . $language->name; ?>"
```

**Comment by infograf768** [comment]
Is RTL OK ?

**Comment by infograf768** [comment]
Tested rtl. Needs to place `$language->name` before the `Text()`
`<?php if ($lang->isRtl()) : ?>` etc.

**Comment by brianteeman** [comment]
Can it be a sprintf?

**Comment by brianteeman** [comment]
Updated with new code to use sprintf so that it doesnt matter what order the verb and the noun are

**Comment by infograf768** [comment]
Looks fine

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on b2eb168ac873a38aa766b129c72a88f99c3eece2<br /><br />Tested successfully in Beta6-dev of 17 December.  RTL not tested.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31655">issues.joomla.org/tracker/joomla-cms/31655</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on b2eb168ac873a38aa766b129c72a88f99c3eece2<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31655">issues.joomla.org/tracker/joomla-cms/31655</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31655">issues.joomla.org/tracker/joomla-cms/31655</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**